### PR TITLE
[FAD-107] - Goldfinch update

### PR DESCRIPTION
**Pull request by luansievers**
### Description

> Merged the code from the latest commit from goldfinch
> Fix borrower artist pools page to display borrow information

### Checklist

- [ ] `CHANGELOG` has been updated (_if appropriate_)
- [ ] Environment variables updated (_if appropriate_)

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -109,28 +109,29 @@ jobs:
...
-      env:
-        cache-name: cache-node-modules
-      with:
-        # npm cache files are stored in `~/.npm` on Linux/macOS
-        path: ~/.npm
-        key: ${{ runner.os }}-build-v0-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-    - run: npx lerna bootstrap --ci --scope @goldfinch-eng/subgraph --include-dependencies
-    - run: npx lerna run build --stream --scope @goldfinch-eng/subgraph --include-dependencies
-    - run: npx lerna run test --stream --scope @goldfinch-eng/subgraph
+  # Disabled until further notice. These tests need a big refactor and dependency upgrade. Arguably a total rewrite.
```
Comment (sachasmart-weavik) [inline .github/workflows/node.js.yml:112]: Lol great.

**Review by sachasmart-weavik — COMMENTED** [review]
Damn, so there weren't any updates to the contracts?

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (luansievers) [reply .github/workflows/node.js.yml:112]: scarry

**Review by v1n33th — APPROVED** [review]

**Comment by sachasmart-weavik** [comment]
Send it.

**Pull request merged**